data/common.yaml

```diff
@@ -5,7 +5,6 @@ ntp::burst: false
 ntp::config_dir: ~
 ntp::config_file_mode: '0644'
 ntp::config: '/etc/ntp.conf'
-ntp::config_template: ~
 ntp::config_epp: ~
 ntp::daemon_extra_opts: ~
 ntp::disable_auth: false
```

Comment:
These default value that are not OS dependent really should be default values in `init.pp` so that puppet-strings can show the actual default value in REFERENCE.md, but it is for another PR.

Reply:
Thanks @smortex , Thanks for the input. Will keep in mind going forward